### ISSUE TITLE
[Feature Request] Ability to "freeze" contents in memory with patch and cheats

### ISSUE THREAD

**Issue by illusion0001** [RPCS3/rpcs3#8606]
I'd like memory freezing to be implemented into the Cheat Manager. It'd make game patches more reliable and easier to create.

Example with standard patching method
```yml
- [ be16, 0x2023193, 0x1337 ] # patches memory with 13 37
```
The value `13 37` would be frozen once it's been applied.

**Comment by illusion0001** [comment]
Regarding the request, PCSX2 has a similar feature
 https://github.com/PCSX2/pcsx2/blob/5f099b1beab87bd278a640e8f07927d218bf7f41/pcsx2/Patch.h#L74

0 for patch once on load (This is what RPCS3 currently has).
1 for continuously patch the address.

**Comment by XLRUUU** [comment]
I'm interested in this feature, I like adding more laps in races in Wipeout HD, but the values has to be changed at each race start, it would be nice if we could freeze those values.

**Comment by gladichl** [comment]
yes, some games don't work with patches without freezing them (aspect ratios, fov etc)

**Comment by illusion0001** [comment]
I think writing a proper PPC patch is better for this

**Comment by XLRUUU** [comment]
> I think writing a proper PPC patch is better for this

This looks much more complicated tho

**Comment by WAZAAAAA0** [comment]
@illusion0001 you closed this issue but has it ever been implemented? 

(I'm trying to convert a little mod I easily made with Cheat Engine into the RPCS3 system, but it's painful and the ability to freeze would make this TRIVIAL)
